src/components/widgets/PAutoComplete.js

```diff
@@ -15,6 +16,7 @@ export default (props) => {
               onPress={() => {
                 props.onSelect && props.onSelect(item)
               }}
+              terms={props.value}
```

Comment:
As I understand, the purpose of passing `terms` to PListItem is highlight the matched part of the item. But that part shouldn't belong to `PListItem`.  PListItem should only render what it needs. Highlighting should be managed in the PAutoComplete component itself